dotCMS/src/main/java/com/dotcms/rendering/velocity/services/PageContextBuilder.java

```diff
@@ -154,6 +146,19 @@ private void populateContext() throws DotDataException, DotSecurityException {
 
     }
 
+    private boolean hasWritePermissionInTemplate(final String templateId) throws DotDataException {
```

Comment:
Why do we need this wrapper method?  We already have the template above.  Just call the permissionapi to see if the user has permission - not recreate a new method.

Reply:
you are right we don't need this wrapper method